cypress/integration/e2e-create-child-journey_spec.js

```diff
@@ -0,0 +1,23 @@
+const { code } = require('../fixtures/mainTypeData.json');
```

Comment:
do we still need this test now? As long as we can test creation of a record of some type (which we do for MainType) there's not much additional value in testing for other types?

Reply:
true we don't need them - they are also covered as part of relationship creation